schema/playground/player_creation_arguments.schema

```diff
@@ -0,0 +1,6 @@
+package playground;
+
+type PlayerCreationParams
```

Comment:
Biggest nit of all time - file name = `player_creation_arguments`, type name is `PlayerCreationParams` 😛